inbm/cloudadapter-agent/cloudadapter/constants.py

```diff
@@ -23,21 +23,23 @@
 
 # Flag for UCC mode to determine which channels to subscribe to
 UCC_FILE = "/etc/intel-manageability/public/ucc_flag"
+UCC_CLIENT_ID = "/etc/ucc/client_id"
```

Comment:
Isn't this client id file?

Reply:
@pillaiabhishek - this PR is still being worked on.  I hadn't updated any changes yet.